proposals/0282-record-dot-syntax.rst

```diff
@@ -68,15 +69,25 @@ For the specification we focus on the changes to the parsing rules, and
 the desugaring, with the belief the type checking and renamer changes
 required are an unambiguous consequences of those.
 
-2.1 ``RecordDotSyntax`` language extension
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+2.1 Language extensions
+~~~~~~~~~~~~~~~~~~~~~~~
 
-This change adds a new language extension ``RecordDotSyntax``.
+This change adds new language extensions ``OverloadedRecordDot`` and
+``OverloadedRecordUpdate``.
+
+The effect of ``OverloadedRecordDot`` is to:
+  - alllow '``.``' in record exprssions;
+  - desguar field access via '``.``'to ``getField`` expressions;
+The effect of ``OverloadedRecordUpdate`` is to:
+  - desguar record updates to ``setField`` expressions.
+
+A record update expression like ``e{lbl₁.lbl₂ = val}`` requires both
+extensions be in effect.
 
 2.1.1 Syntax
 ^^^^^^^^^^^^
 
-In the event the language extension is enabled:
+In the event the language extensions are enabled:
```

Comment:
Which extensions?  There are two

Reply:
- Changed to "In the event the language extensions ``OverloadedRecordDot` and `OverloadedRecordUpdate` are enabled"